test/runtests.jl

```diff
@@ -156,5 +157,13 @@ end
     @test ts_reporting_copy.results == ts_reporting.results
 end
 
+@testset "showerror" begin
+    @test_throws TestReports.PkgTestError throw(TestReports.PkgTestError("Test"))
+    io = IOBuffer()
+    showerror(io, TestReports.PkgTestError("Error text"), "")
+    @test String(take!(io)) == "Error text"
```

Comment:
i think this can be done with `sprint` so don't need the `IOBuffer`

Reply:
Thanks, good suggestion